bitcoin/src/crypto/key.rs

```diff
@@ -8,7 +8,6 @@
 use core::fmt::{self, Write as _};
 use core::ops;
 use core::str::FromStr;
-
```

Comment:
This line should not be removed mate.

Reply:
Oh, so the dependencies are also seperated. Thanks!